data/templates/dovecot/dovecot.conf

```diff
@@ -12,10 +12,25 @@ protocols = imap sieve {% if pop3_enabled == "True" %}pop3{% endif %}
 
 mail_plugins = $mail_plugins quota
 
-ssl = yes
+###############################################################################
+
+# generated 2020-04-03, Mozilla Guideline v5.4, Dovecot 2.2.27, OpenSSL 1.1.1l, intermediate configuration
+# https://ssl-config.mozilla.org/#server=dovecot&version=2.2.27&config=intermediate&openssl=1.1.1l&guideline=5.4
+
+ssl = required
+
 ssl_cert = </etc/yunohost/certs/{{ main_domain }}/crt.pem
 ssl_key = </etc/yunohost/certs/{{ main_domain }}/key.pem
-ssl_protocols = !SSLv3
+
+ssl_dh_parameters_length = 2048
+
+# intermediate configuration
+ssl_protocols = TLSv1.2 TLSv1.3
```

Comment:
Are you sure that TLSv1.3 is already supported on buster ?

Reply:
I was suspicious too but that's indeed what Mozilla recommends with Dovecot 2.2.27 (= on Stretch) and tweaking the conf with this, Dovecot still works (or at least the service starts...)